src/harness/testcases/configs/test_WINNF_FT_S_MCP_1/sas_hareness.config

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is this used?

Reply:
updated,   included in the default.config